easybuild/easyconfigs/s/scikit-build/scikit-build-0.10.0-foss-2020a-rc3-Python-3.8.2.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'scikit-build'
+version = '0.10.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://scikit-build.readthedocs.io/en/latest'
+description = """Scikit-Build, or skbuild, is an improved build system generator
+for CPython C/C++/Fortran/Cython extensions."""
+
+toolchain = {'name': 'foss', 'version': '2020a-rc3'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('packaging', '20.3', {
+        'checksums': ['3c292b474fda1671ec57d46d739d072bfd495a4f51ad01a055121d81e952b7a3'],
+    }),
+    (name, version, {
+        'modulename': 'skbuild',
+        'checksums': ['7342017cc82dd6178e3b19377389b8a8d1f8b429d9cdb315cfb1094e34a0f526'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
```

Comment:
This can be dropped, it's the same as the default for both `PythonPackage` and `PythonBundle`.